src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/SetAzurePolicySetDefinition.cs

```diff
@@ -20,64 +20,69 @@ namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation
+        [Parameter(ParameterSetName = PolicyCmdletBase.IdParameterSet, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.SetPolicySetDefinitionIdHelp)]
         [ValidateNotNullOrEmpty]
         public string Id { get; set; }
 
         /// <summary>
         /// Gets or sets the policy set definition display name parameter
         /// </summary>
-        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The display name for policy set definition.")]
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.SetPolicySetDefinitionDisplayNameHelp)]
         [ValidateNotNullOrEmpty]
         public string DisplayName { get; set; }
 
         /// <summary>
         /// Gets or sets the policy set definition description parameter
         /// </summary>
-        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The description for policy set definition.")]
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.SetPolicySetDefinitionDescriptionHelp)]
         [ValidateNotNullOrEmpty]
         public string Description { get; set; }
 
         /// <summary>
         /// Gets or sets the policy definition parameter
         /// </summary>
-        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The policy set definition. This can either be a path to a file name containing the policy definitions, or the policy set definition as string.")]
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.SetPolicySetDefinitionPolicyDefinitionHelp)]
         [ValidateNotNullOrEmpty]
         public string PolicyDefinition { get; set; }
 
+        /// <summary>
+        /// Gets or sets the policy set definition management group name parameter.
+        /// </summary>
+        [Parameter(ParameterSetName = PolicyCmdletBase.ManagementGroupNameParameterSet, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.SetPolicySetDefinitionManagementGroupHelp)]
+        [ValidateNotNullOrEmpty]
+        public string ManagementGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the policy set definition subscription id parameter.
+        /// </summary>
+        [Parameter(ParameterSetName = PolicyCmdletBase.SubscriptionIdParameterSet, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.SetPolicySetDefinitionSubscriptionIdHelp)]
+        [ValidateNotNullOrEmpty]
+        public Guid? SubscriptionId { get; set; }
+
         /// <summary>
         /// Executes the cmdlet.
```

Comment:
Should support updating metadata like the policyDefinition cmdlet does

Reply:
Also parameters

Reply:
Done